File: app/src/main/java/com/example/rodrigo/sgame/CommonGame/Note.java
package com.example.rodrigo.sgame.CommonGame;

public class Note {
    public byte noteType = 0;
    boolean sudden = false;
    boolean hidden = false;
    public boolean fake = false;


    public Note(byte note){
        noteType= note;

    }

    public Note() {

    }
}

File: app/src/main/java/com/kyadevs/stepdroid/CommonGame/Note.kt
package com.kyadevs.stepdroid.CommonGame

class Note {
    var noteType: Byte = 0
    internal var sudden = false
    internal var hidden = false
    var fake = false


    constructor(note: Byte) {
        noteType = note

    }

    constructor() {

    }
}
